src/activemasternode.cpp

```diff
@@ -86,9 +79,14 @@ void CActiveMasternode::ManageStatus()
             LogPrintf("CActiveMasternode::ManageStatus() - not capable: %s\n", notCapableReason);
             return;
         }
+
+        notCapableReason = "Waiting for start message from controller.";
+        LogPrintf("%s - not capable: %s : %s\n", __func__ , notCapableReason, errorMessage);
```

Comment:
No need  to log `errorMessage` here, as that is only related to `CheckDefaultPort`.

Reply:
done